analytics/docker-compose.yml

```diff
@@ -26,5 +26,14 @@ services:
     depends_on:
       - grants-analytics-db
 
+  grants-metabase:
+    image: metabase/metabase:latest
+    container_name: grants-metabase
+    volumes:
+    - /dev/urandom:/dev/random:ro
+    ports:
+      - 3000:3000
```

Comment:
The frontend container exposes 3000. Can we switch to something like `3050` or `3333`?

Reply:
@acouch I changed it to 3100